Getting-Started.adoc

```diff
@@ -39,29 +39,35 @@ helm init --service-account=tiller
 
 NOTE: To verify that the Tiller pod has started execute the following command: `kubectl get pod --namespace kube-system` and you should see the `tiller` pod running.
 
-==== Add the Sk8s repository to your Helm environment
+==== Add the Riff repository to your Helm environment
 
-Run the following commands to add the `sk8s-incubator` repository:
+Run the following commands to add the repository:
 
 ----
-helm repo add sk8s-incubator http://sk8s-incubator.cfapps.io
-helm repo update
+helm repo add riffrepo https://riff_charts.storage.googleapis.com
+helm repo update riffrepo
 ----
 
-==== Install Sk8s FaaS helm chart
+==== Install Riff FaaS helm chart
 
-Install the Sk8s Helm chart using one of the following commands.
+Install the Riff Helm chart as follows:
+
+- `helm search riffrepo/riff` and identify the version you're looking to install (use latest if unsure)
+- Download the starter install script: `curl -sL https://riff_charts.storage.googleapis.com/riff-<version>-install-example.sh > riff_install.sh`
+- Ensure the script has execute permissions: `chmod +x riff_install.sh`
+
+*Note:* the starter script above contains references to Docker image tags appropriate for the version of the Helm chart being deployed. If a value needs to be overridden, arguments to the script can be passed in the form `--set key=value`
 
 For a cluster that does not have RBAC enabled use:
 
 ----
-helm install --name demo sk8s-incubator/sk8s
+./riff_install.sh riffrepo/riff --name=riff
```

Comment:
I don't think we want to show "riff" as the example name since that leads to all of the components being named `riff-riff-<componentname>`, but maybe we want to reconsider that naming strategy (not sure if it's a helm idiom @trisberg ?)... an example from the chart templates:

```
metadata:
  name: {{ template "fullname" . }}-function-controller
```